src/utils/AppConsts.js

```diff
@@ -12,20 +12,20 @@ const PROD = 'http://api.loom.digital';
 const JSON = 'json';
 const CSV = 'csv';
 
-const HIDDEN = 'hidden';
-const DISCOVER = 'discover';
-const READ = 'read';
-const WRITE = 'write';
-const OWNER = 'owner';
+const HIDDEN = 'HIDDEN';
```

Comment:
PR for splitting AppConsts into multiple files: https://github.com/kryptnostic/gallery/pull/40